Templates/Databases/MicrosoftSQL/CSV file to MS SQL Database insert/long-description.md

```diff
@@ -0,0 +1,29 @@
+This process will read a **CSV file** which contains key-value pairs representing the data to be insert into the database from an **SFTP server**. 
+
+
+
+![Template](assets/CSV file to MS SQL Database insert.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The SFTP server user should have the permissions to connect and access 
+  the files that Frends needs to download.
```

Comment:
the -> The.
Also, we are only talking about one file here, so singular should be here. this could say "The CSV file that Frends needs to download exists on the SFTP server"